woocommerce/woocommerce-gutenberg-products-block
# Product Hero Pattern > Adjust the content width

## What

Minor update to the Product Hero Pattern to address design feedback, ensuring the width of the columns is proportional (40%-60% and the `contentSize` is 650px).

Fixes #10637

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new post
2. Insert the Product Hero pattern
3. Make sure the pattern is properly displayed both in the editor and on the front end


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|  <img width="1199" alt="Screenshot 2023-09-04 at 22 04 40" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/ffc2c7a9-00b3-4035-b86a-690d572bab40"> | <img width="1234" alt="Screenshot 2023-09-04 at 22 04 22" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/4e8c4c24-d96b-44bd-93b2-b69203dc41ff"> |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Updated the content width for the Product Hero pattern.
